src/linker/configuration.py

```diff
@@ -87,6 +87,7 @@ def _validate(self) -> None:
         errors = {
             **self._validate_files(),
             **self._validate_input_data(),
+            **self.schema.validate_input_filenames(self.input_data),
```

Comment:
Oh, did you move `add_input_filename_bindings` outside of `_get_schema` so that you validate the input filenames first?

Reply:
correct